Source/CoreDataFiles/CoreDataAppDelegate.swift

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
This is essentially a wrapper that the app delegate in the main project must conform to. It just allows the dev pod to make use of the app delegate without needing to depend on the main project